migrations/01-create-artist-table.sql

```diff
@@ -0,0 +1,5 @@
+  CREATE TABLE Artists (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  genre VARCHAR(255) NOT NULL
+  );
```

Comment:
Id put some indentation into all these sql scripts, it just makes them easier to read and is a good habit to get into
```
CREATE TABLE Artists (
  id SERIAL PRIMARY KEY,
  name VARCHAR(255) NOT NULL,
  genre VARCHAR(255) NOT NULL
);
```